ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/policy-maps-copp.j2

```diff
@@ -0,0 +1,22 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - policy map copp #}
+{% if policy_maps.copp is arista.avd.defined %}
+{%     if policy_maps.copp.name is arista.avd.defined %}
```

Comment:
This check can be collapsed into just checking for copp.name.

Reply:
Fixed.